brickflow_plugins/databricks/sla_sensor.py

```diff
@@ -0,0 +1,596 @@
+from pydantic import SecretStr, BaseModel, ValidationError
+from typing import Union, List, Optional
+from datetime import timedelta, datetime, timezone
+from warnings import warn
+from textwrap import dedent
+import requests
+import time
+import json
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import smtplib
+from brickflow import ctx
+from brickflow_plugins.databricks.workflow_dependency_sensor import (
+    WorkflowTaskDependencySensor,
+    WorkflowDependencySensorException,
+)
+
+
+class EmailParams(BaseModel):
+    email_list: str
```

Comment:
As we have the pydantic model now, probably it's easier for users to provide a list and we handle the rest.
Same goes for - cc as well.

If it is string and comma separated - after splitting, we also need to handle strip()

```suggestion
    email_list: List[str]
```

Let me know your thoughts, other wise the rest of the pr looks good to me

Reply:
That makes sense to me. Also is a little more intuitive from an end-user perspective. I'll make those updates